query-engine/connector-test-kit/src/test/scala/util/ConnectorConfig.scala

```diff
@@ -21,8 +24,15 @@ case class ConnectorConfig(
 object ConnectorConfig {
   lazy val instance: ConnectorConfig = {
     val filePath = EnvVars.serverRoot + "/current_connector"
+
+    val charset = if (EnvVars.isWindows) {
+      StandardCharsets.UTF_16
+    } else {
+      StandardCharsets.UTF_8
+    }
```

Comment:
The might not be needed. I would try `Charset.defaultCharset()` and see if this works as well.

Reply:
Uh. Yes.